src/riak_repl2_fscoordinator.erl

```
@@ -462,7 +460,6 @@ handle_info(send_next_whereis_req, State) ->
     {noreply, State2};
 
 handle_info(_Info, State) ->
-    lager:info("ignoring ~p", [_Info]),
```

I say let it crash or log it. just my 2c.